{{cookiecutter.project_name}}/.pre-commit-config.yaml

```diff
@@ -7,9 +7,13 @@ default_stages:
 minimum_pre_commit_version: 2.16.0
 repos:
     - repo: https://github.com/psf/black
-      rev: 22.3.0
+      rev: "22.3.0"
```

Comment:
It's good practice to ensure version number are a string. 
I ran recently into that when updating to Python 3.10. Yaml thinks its a float an Python 3.1 gets installed.

Reply:
Ah, YAML.

So:

* What does pre-commit autoupdate think about this?
* Should all of the versions be quoted?
* Should this be a `prettier` pre-commit check? (Could be a separate PR)

Reply:
> What does pre-commit autoupdate think about this?

Last time I checked, `pre-commit autoupdate` keeps quotes.

> Should all of the versions be quoted?

We can quote `vX.Y` style tags for stylistic/consistency reasons, but the leadin “v” always makes YAML see it as a string.

> Should this be a `prettier` pre-commit check? (Could be a separate PR)

Is that possible? If so, it might be a cool idea for a separate PR!